DEPRECATION.md

```diff
@@ -85,75 +78,130 @@ here][link-enable-https].
 [link-enable-https]:
   https://cube.dev/docs/deployment/production-checklist#enable-https
 
-### `hearBeatInterval`
-
-**Deprecated in Release: v0.23.8**
-
-This option for [`@cubejs-client/ws-transport`][link-hearbeatinterval] has been
-replaced by `heartBeatInterval`.
-
-[link-hearbeatinterval]:
-  https://cube.dev/docs/@cubejs-client-ws-transport#web-socket-transport-hear-beat-interval
+### Embedding Cube.js within Express
 
-### Node.js 8
+**Deprecated in Release: v0.24.0**
 
-**Deprecated in Release: v0.22.4**
+Embedding Cube.js into Express applications is deprecated due to performance and
+reliability considerations. [Read more about this change
```

Comment:
```suggestion
reliability issues. [Read more about this change
```

Reply:
Is "issues" too strong?

Reply:
Possibly ... this PR is specifically related to changes for v0.26 though 👍